src/ert/_c_wrappers/enkf/ert_workflow_list.py

```diff
@@ -13,51 +13,59 @@
 class ErtWorkflowList:
     def __init__(
         self,
-        content_dict=None,
-        config_dict=None,
+        workflow_job_info: list = None,
+        workflow_job_dir_info: list = None,
+        hook_workflow_info: list = None,
+        workflow_info: list = None,
     ):
-        if content_dict is None and config_dict is None:
-            raise ValueError(
-                "Failed to construct ErtWorkflowList instance with no config object"
-            )
-
-        if content_dict is not None and config_dict is not None:
-            raise ValueError(
-                "Failed to construct ErtWorkflowList "
-                "instance with multiple config object"
-            )
-
         self._workflow_jobs = {}
         self._workflow = {}
         self._hook_workflow_list = []
         self._workflow_joblist = WorkflowJoblist()
         self._parser = WorkflowJob.configParser()
 
-        self._workflow_job_info = []
-        self._workflow_job_dir_info = []
-        self._workflow_info = []
-        self._hook_workflow_info = []
-
-        if content_dict is not None:
-            self._handle_config_content_dict(content_dict)
-
-        if config_dict is not None:
-            self._handle_config_dict(config_dict)
-
-        for workflow_job in self._workflow_job_info:
+        for workflow_job in workflow_job_info:
```

Comment:
This will not work with the default value `None`

Reply:
Gotcha. :+1: